Samples/PublicSamples/RecordingBot/docs/tutorials/deploy/aks.md

```diff
@@ -0,0 +1,719 @@
+
+To check now if our system nodepool `nodepool1` is tainted check the _nodeTaints_ field of the nodepool. If the field has the value of `null` we can continue with [setting the DNS name](#set-dns-name). If it's not the case, like in our case, we have to untaint the nodepool. If your system nodepool has a different name replace `nodepool1` with the name you see in your previous result.
+
+```powershell
+az aks nodepool update 
+    --cluster-name recordingbotcluster
+    --name nodepool1
+    --resource-group recordingbottutorial
+    --subscription "recordingbotsubscription"
+    --%
+    --node-taints ""
+```
+
+The result of the command should look similar to:
+
+```json
+{
+  "availabilityZones": null,
+  "capacityReservationGroupId": null,
+  "count": 1,
+  "creationData": null,
+  "currentOrchestratorVersion": "1.28.5",
+  "enableAutoScaling": false,
+  "enableEncryptionAtHost": false,
+  "enableFips": false,
+  "enableNodePublicIp": false,
+  "enableUltraSsd": false,
+  "gpuInstanceProfile": null,
+  "hostGroupId": null,
+  "id": "/subscriptions/yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyyy/resourcegroups/recordingbottutorial/providers/Microsoft.ContainerService/managedClusters/recordingbotcluster/agentPools/nodepool1",
+  "kubeletConfig": null,
+  "kubeletDiskType": "OS",
+  "linuxOsConfig": null,
+  "maxCount": null,
+  "maxPods": 30,
+  "minCount": null,
+  "mode": "System",
+  "name": "nodepool1",
+  "networkProfile": null,
+  "nodeImageVersion": "AKSUbuntu-2204gen2containerd-202403.25.0",
+  "nodeLabels": null,
+  "nodePublicIpPrefixId": null,
+  "nodeTaints": null,
+  "orchestratorVersion": "1.28.5",
+  "osDiskSizeGb": 128,
+  "osDiskType": "Managed",
+  "osSku": "Ubuntu",
+  "osType": "Linux",
+  "podSubnetId": null,
+  "powerState": {
+    "code": "Running"
+  },
+  "provisioningState": "Succeeded",
+  "proximityPlacementGroupId": null,
+  "resourceGroup": "recordingbottutorial",
+  "scaleDownMode": null,
+  "scaleSetEvictionPolicy": null,
+  "scaleSetPriority": null,
+  "spotMaxPrice": null,
+  "tags": null,
+  "type": "Microsoft.ContainerService/managedClusters/agentPools",
+  "typePropertiesType": "VirtualMachineScaleSets",
+  "upgradeSettings": {
+    "drainTimeoutInMinutes": null,
+    "maxSurge": "10%",
+    "nodeSoakDurationInMinutes": null
+  },
+  "vmSize": "standard_d2s_v3",
+  "vnetSubnetId": null,
+  "workloadRuntime": null
+}
+```
+
+As you can see the _nodeTaints_ field now updated to `null`.
+
+## Set DNS name
+
+When we created our AKS cluster, we automatically created a public IP too. Now we need to create a DNS name for the public IP. The resouce of the IP is not in our default resource group `recordingbottutorial`, because the creation of our AKS cluster created a Resource group with the managed resources. If you already know the resource group name of the managed resources, you can coninue with [getting the public IP resource name](#get-the-public-ip-resource-name).

```

Comment:
```suggestion
When we created our AKS cluster, a public IP was automatically created for us. To use this IP Address, we want to create a DNS entry so we can reference it by name. Since the IP Address was automatically created for us, it resides in the automatically created resource group which is an Azure Managed Resource. If you have not noted down the name of this resource group in [Create Azure Kubernetes Cluster](#create-azure-kubernetes-Cluster), follow the next step to find out the name. Otherwise you can skip ahead to [getting the public IP resource name](#get-the-public-ip-resource-name).

```